src/Libraries/Nop.Services/Connected Services/EuropaCheckVatService/Reference.cs

```diff
@@ -325,7 +325,7 @@ public virtual System.Threading.Tasks.Task OpenAsync()
             return System.Threading.Tasks.Task.Factory.FromAsync(((System.ServiceModel.ICommunicationObject)(this)).BeginOpen(null, null), new System.Action<System.IAsyncResult>(((System.ServiceModel.ICommunicationObject)(this)).EndOpen));
         }
         
-        public virtual System.Threading.Tasks.Task CloseAsync()
```

Comment:
Hi, @jafin. Thanks for the help. For the future, please note that this class is auto-generated, and all the changes you made to it will be overwritten with the next update, so there is no point in making anything manually to it

Reply:
@skoshelev would it only regenerate when the web service is manually refreshed?
Did you still want to accept the Patternmode changes for redis?  I can revert the Service reference change?

Reply:
@jafin Thank you, yes, of course, although the file is auto-generated, it will only be updated when the service is manually updated, however, it will be almost impossible to track changes in it